packages/strapi-plugin-graphql/services/type-definitions.js

```diff
@@ -66,7 +68,7 @@ const buildTypeDefObj = model => {
     });
 
   // Remove private attributes defined per model or globally
-  const privateAttributes = getPrivateAttributes(model);
+  const privateAttributes = contentTypes.getPrivateAttributes(model);
```

Comment:
I think this forEach loop to delete the fields will not be necessary anymore as you filter them beforehand